site/viamguides/src/environment-variables/environment-variables.md

```diff
@@ -0,0 +1,223 @@
+
+- How to create a Python virtual environment
+- How to store and reuse values as Python variables
+- How to move control code to your machine
+- How to use variables during development and in production
+
+### What You’ll Need
+
+- A computer with MacOS, Windows, or Linux 
+- [Python3](https://www.python.org/downloads/)
+- [VS Code](https://code.visualstudio.com/download) installed, or another similar code editor of your choice.
+
+### What You'll Build
+
+- An example project, such as a Viam rover, using Python variables to access our machine in Viam.
+
+<!-- ------------------------ -->
+## Set up a machine for local development
+Duration: 2
+
+### Borrow a rover online with Try Viam
+
+> aside positive
+> Skip this step if you already have a machine set up in Viam, and want to use variables within your own project.
+
+1. **Borrow a Viam rover**: Let's use [Try Viam](https://app.viam.com/try) to borrow a rover online at no cost which is already configured with all the components you need. If you want to use your own wheeled robot, see more details in [this tutorial](https://docs.viam.com/get-started/drive-rover/).
+ ![Borrow a rover with Try Viam](assets/tryViam.png)
+1. Once the rover is configured, click **Try Viam Now**.
+ ![Try Viam now](assets/tryViamNow.png)
+1. Examine the modular resources that make up the machine in the Viam app listed in the left sidebar of the **CONFIGURE** and **CONTROL** tabs.
+ ![Viam app](assets/ViamRover.png)
+
+<!-- ------------------------ -->
+## Prepare a virtual environment
+Duration: 2
+
+### Prepare a project directory
+
+1. **Set up your project directory**: From the command line in your terminal window, make a directory for your project, for example, called `viam-rover`. And navigate into the directory.
+    ```bash
+    mkdir viam-rover
+    cd viam-rover
+    ```
+
+### Prepare a virtual environment
+[Set up a Python virtual environment](https://docs.viam.com/sdks/python/python-venv/) to avoid any conflicts with other projects or your system.
+1. From the command line in your terminal window, install [`virtualenv` using `pip`](https://virtualenv.pypa.io/en/latest/installation.html#via-pip)
+    ```bash
+    python3 -m pip install --user virtualenv
+    ```
+1. **Create and activate a virtual environment**: In the project directory `viam-rover`, create and activate a virtual environment called `.venv` for Python to run in.
+    ```bash
+    python3 -m venv .venv
+    source .venv/bin/activate
+    ```
+Now, `.venv` prepends the commands in your terminal window to indicate the Python packages being used are from this particular environment. You can exit this environment by running `deactivate`.
+
+### Install the Python SDK
+
+Install the Viam Python SDK and all required general dependencies.
+
+1. **Install the SDK**: From the command line in your terminal window (inside the activated `.venv` Python environment), install the Viam Python SDK:
+    ```bash
+    pip3 install viam-sdk
```

Comment:
Once inside an activated virtual environment, it is typical to use the standard `pip` or `python` commands rather than the `pip3` or `python3` versions. 

```suggestion
    pip install viam-sdk
```